src/raster/r.in.nasadem/r.in.nasadem.py

```diff
@@ -347,7 +347,7 @@ def createTMPlocation(epsg=4326):
     else:
         currepsg = proj["srid"].split("EPSG:")[1]
 
-    currepsg = ":".join(srid.split(":")[-1:])
+    currepsg = ":".join(proj["srid"].split(":")[-1:])
```

Comment:
Just delete this row altogether

Reply:
Done in 12d8de57692c4924bf15609331e79e2ae31ffb0c